inc/agent.class.php

```diff
@@ -1982,6 +1982,10 @@ public static function getSpecificValueToDisplay($field, $values, array $options
                return $output;
             }
             break;
+         case 'mdm_type':
```

Comment:
This code will transform the field for API consumers (the dashboard). I think we should keep the data as is in such case, and delegate the dashboard to translate the data for display. isAPI() is your friend :+1: